WalletWasabi.Gui/Controls/WalletExplorer/WalletAdvancedViewModel.cs

```diff
@@ -25,7 +25,9 @@ public WalletAdvancedViewModel(WalletViewModel walletViewModel) : base(walletVie
 		{
 			Items = new ObservableCollection<WalletActionViewModel>();
 
+#pragma warning disable IDE0053 // Use expression body for lambda expressions
 			ExpandItCommand = ReactiveCommand.Create(() => { IsExpanded = !IsExpanded; });
```

Comment:
Looks like it's the same as the other, why add pragma warning?

Reply:
On this one if you use an expression body you get a cast error.

Reply:
![Capture](https://user-images.githubusercontent.com/52379387/72667166-032b8a80-3a19-11ea-8046-31375be88b4f.PNG)

Reply:
Indeed, I knew I solved it before, but I didn't remember how. Just change the declaration of `ExpandItCommand` and the compiler will be happy.

Reply:
Changing `ExpandItCommand` from:
`public ReactiveCommand<Unit, Unit> ExpandItCommand { get; }`
to:
`public ReactiveCommand<Unit, bool> ExpandItCommand { get; }`
makes the compiler happy.

Is there no problems in doing so?
If there isn't then I should do this where we used the pragma warning.

Reply:
Correct.

Reply:
done.